en/advanced/parameters_and_configurations.md

```diff
@@ -164,7 +173,7 @@ orb_unsubscribe(parameter_update_sub);
 ```
 
 Call `parameters_update(parameter_update_sub);` periodically in code to check if there has been an update (this is boilerplate):
-```cpp
+```cpp 
```

Comment:
Does anything need to change here? The application template looks a bit different:
```
void TemplateModule::parameters_update(bool force)
{
	// check for parameter updates
	if (_parameter_update_sub.updated() || force) {
		// clear update
		parameter_update_s update;
		_parameter_update_sub.copy(&update);

		// update parameters from storage
		updateParams();
	}
}
```
The app temp also has a "run" method that is not explained https://github.com/PX4/PX4-Autopilot/blob/master/src/templates/template_module/template_module.cpp#L138